pynumdiff/utils/utility.py

```diff
@@ -28,18 +28,19 @@ def huber_const(M):
     return np.sqrt((2*a*(1 + M**2)/M**2 + b)/(a + b))
 
 
-def integrate_dxdt_hat(dxdt_hat, dt_or_t):
+def integrate_dxdt_hat(dxdt_hat, dt_or_t, axis=0):
     """Wrapper for scipy.integrate.cumulative_trapezoid. Use 0 as first value so lengths match, see #88.
 
     :param np.array[float] dxdt_hat: estimate derivative of timeseries
     :param float dt_or_t: step size if given as a scalar or a vector of sample locations
+    :param int axis: data dimension along which to integrate
 
     :return: **x_hat** (np.array[float]) -- integral of dxdt_hat
     """
-    return cumulative_trapezoid(dxdt_hat, initial=0)*dt_or_t if np.isscalar(dt_or_t) \
-            else cumulative_trapezoid(dxdt_hat, x=dt_or_t, initial=0)
+    return cumulative_trapezoid(dxdt_hat, initial=0, axis=axis)*dt_or_t if np.isscalar(dt_or_t) \
```

Comment:
`scipy` is kind enough to give us an `axis` as well. How courteous of them. We should be courteous too. Hence #76 and this PR.